src/image/filters.js

```diff
@@ -140,10 +140,9 @@ Filters._createImageData = function (width, height) {
  * use case of creating a filter chain that executes without actually updating
  * the canvas in between everystep.
  *
- * @param  {[type]} func   [description]
- * @param  {[type]} canvas [description]
- * @param  {[type]} level  [description]
- * @return {[type]}        [description]
+ * @param  {HTMLCanvasElement} canvas [description]
+ * @param  {function(ImageData,Object)} func   [description]
```

Comment:
i would recommend keeping the metadata for as long as possible, some tools may be able to use it, others may need to be fixed...